common-npm-packages/azure-arm-rest-v2/azureAppServiceUtility.ts

```diff
@@ -114,6 +114,15 @@ export class AzureAppServiceUtility {
         if(scmPolicyCheck === false) {
             token = await this._appService._client.getCredentials().getToken();
             method = "Bearer";
+            // Though bearer AuthN is used, lets try to set publish profile password for mask hints to maintain compat with old behavior for MSDEPLOY. 
+            // This needs to be cleaned up once MSDEPLOY suppport is reomve. Safe handle the exception setting up mask hint as we dont want to fail here.
```

Comment:
Changing a common library is a multistep process.  Version in the library needs to be bumped.  Then merged to generate npm package.  Then dependent tasks need to be updated to update version reference in package.json and package-lock.json (Generated by build)

Reply:
Oh I see.. I was wondering why my changes did not get picked up. understood now.